client/desktop/lib/window-manager/index.js

```diff
@@ -15,6 +15,9 @@ const path = require( 'path' );
 // electron.
 let screen;
 app.on( 'ready', () => {
+	if ( process.platform === 'win32' ) {
+		app.setAppUserModelId( 'com.automattic.wordpress' );
```

Comment:
Can you add a comment as to what this does?

Reply:
Added comment in 945982702ddf58845ff681784dc78132c0a82906